### PR TITLE
feat: integrate Chapter and Lesson modules into SCORM processing

### PR DESCRIPTION
**Pull request by typeWolffo**
## Jira issue(s)
[LC-489](https://selleolabs.atlassian.net/browse/LC-489)

## Overview
- now, during scorm processing the chapters are generated as in every other course

## Screenshots
image

[LC-489]: https://selleolabs.atlassian.net/browse/LC-489?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: apps/api/src/scorm/services/scorm.service.ts [hunk 1]
```diff
@@ -362,6 +413,77 @@ export class ScormService {
...
+          type: resource.$.type,
+          scormtype: resource.$["adlcp:scormtype"],
+        },
+      ]),
+    );
+
+    const items = Array.isArray(organization.item) ? organization.item : [organization.item];
+
+    return items.map((chapterItem: any, chapterIndex: number) => {
+      const subItems = chapterItem.item
```
Comment (typeWolffo) [inline apps/api/src/scorm/services/scorm.service.ts:434]: will be simplified in next PR

File: apps/api/src/scorm/services/scorm.service.ts [hunk 2]
```diff
@@ -362,6 +413,77 @@ export class ScormService {
...
+        const resource = resourceMap.get(resourceId);
+        const lessonTitle = Array.isArray(lessonItem.title)
+          ? lessonItem.title[0]
+          : lessonItem.title;
+        const isQuiz = lessonTitle.toLowerCase().includes("quiz");
+
+        return {
+          title: lessonTitle,
+          identifier: lessonItem.$.identifier,
+          // @ts-expect-error tet
```
Comment (typeWolffo) [inline apps/api/src/scorm/services/scorm.service.ts:459]: next PR

File: apps/api/src/scorm/services/scorm.service.ts [hunk 3]
```diff
@@ -4,24 +4,46 @@ import path from "path";
...
+import { AdminLessonService } from "src/lesson/adminLesson.service";
+import { LESSON_TYPES } from "src/lesson/lesson.type";
 import { S3Service } from "src/s3/s3.service";
 
 import { SCORM } from "../constants/scorm.consts";
 import { ScormRepository } from "../repositories/scorm.repository";
 
 import type { UUIDType } from "src/common";
 
+type ScormChapter = {
```
Comment (wielopolski) [inline apps/api/src/scorm/services/scorm.service.ts:22]: In next PR extract it to separate file

**Review by wielopolski — APPROVED** [review]
LG 🍻

**Review by piotr-pajak — APPROVED** [review]

**Pull request merged**